docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
+![Create Alias screen](../../../.gitbook/assets/aws-enter-preferred-alias.png)
+
+4. Navigate to the [Request certificate page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/request). Keep the default "Request a public certificate" option selected and click the "Next" button.
+![Request public certificate page](../../../.gitbook/assets/aws-request-public-cert.png)
+
+5. Input the certificate details and click "Request":
+    - Fully qualified domain name: Input `<your-dev-account-name>.civiform.dev` (eg. `jdoe-dev.civiform.dev`)
+    - Validation method: Leave the default selection "DNS validation - recommended" selected.
+    - Key algorithm: Leave the default selection "RSA 2048" selected.
+![Enter certificate details page](../../../.gitbook/assets/aws-enter-cert-details-page.png)
+
+6. You should be redirected to the [Certificates page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/list) and there should be a banner at the top of the page that says "Successfully requested certificate with ID ...". Click "View certificate". This will take you to the certificate details page.
+![View certificate banner](../../../.gitbook/assets/aws-view-cert-banner.png)
+
+7. In the "Domains" section of the certificate details page, there should be a single row with `CNAME` listed under the "Type" column. The values in the "CNAME name" and "CNAME value" columns are used to validate that we control the domain (`civiform.dev`) requested by the certificate. Copy both the "CNAME name" and "CNAME value" column values somewhere for yourself temporarily, you will need them in the following steps.
+![Certificate details page](../../../.gitbook/assets/aws-copy-cname-values.png)
+
+8. In a new tab, open the [civiform.dev DNS configuration page](https://domains.google.com/registrar/civiform.dev/dns) and click "Manage custom records".
+![Manage civiform.dev dns records](../../../.gitbook/assets/dns-manage-custom-records.png)
+9. Scroll to the bottom of this list and click "Create new record" (note: the UI is a little buggy you might have to click this twice to get an empty row to appear).
+![Create new dns record](../../../.gitbook/assets/dns-create-new-record.png)
+
+10. Input the following details into the new row and click "Save":
+    - Host name: Input the value of the "CNAME name" column you copied in step 7 above. Remove the `civiform.dev` part at the end of the string. Google Domains automatically appends this to whatever you enter. If you do not remove it, you will add a CNAME record for `<random_id>.<dev-account>.civiform.dev.civiform.dev` which is incorrect.
+    - Type: Select "CNAME"
+    - TTL: Leave the default value "3600"
+    - Data: Input the value of the "CNAME value" column you copied in step 7 above.
+![Save new record details](../../../.gitbook/assets/dns-new-record-details.png)
```

Comment:
Make sure the Host name does NOT include `.civiform.dev`, since this is what AWS will automatically copy to the clipboard, but Google Domains fills this part in for you.

Reply:
Oh nevermind, you already have this! I just misread the text in step 10 :)